scrapy/logformatter.py

```diff
@@ -29,6 +29,9 @@ class LogFormatter(object):
     *   ``args`` should be a tuple or dict with the formatting placeholders for ``msg``.
         The final log message is computed as ``msg % args``.
 
+    Users can define their own ``LogFormatter`` class if they want to customise how
+    each action is logged or if they want to omit it entirely. In order to omit
+    logging an action the method must return ``None``.
```

Comment:
:lipstick: What about adding an empty line at the end, to make the new text a paragraph of its own instead of part of the existing paragraph?

Reply:
Makes sense, updated in e5a5ecfd